dev-libs/univalue/univalue-1.0.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit eutils
+
+DESCRIPTION="C++ universal value object and JSON library"
+HOMEPAGE="https://github.com/jgarzik/univalue"
+LICENSE="MIT"
+
+SRC_URI="https://codeload.github.com/jgarzik/${PN}/tar.gz/v${PV} -> ${P}.tgz"
+SLOT="0/0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	./autogen.sh || die
```

Comment:
what does this do? bootstrap autotools?

Reply:
I suppose what @SoapGentoo was getting at, is that you're supposed to use `eautoreconf` for this.